src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj

```diff
@@ -42,6 +42,23 @@ Fixed issue #447.
 Fixed issue #449.
     </PackageReleaseNotes>
   </PropertyGroup>
+  <!--We manually configure LanguageTargets for Xamarin due to .Net SDK TFMs limitation https://github.com/dotnet/sdk/issues/491 -->
```

Comment:
Comment not applicable to this line.

Acceptance of this PR will kick of a preview build. If you expect that we will release this, I suggest that you add package release notes.

Reply:
Also note that commit messages are automatically used for specifying the GitHub release so we should either clean up the commit messages, or squash them into a single message.
https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/releases